locals.tf

```
@@ -688,6 +696,10 @@ controller:
   hetzner_ccm_values = var.hetzner_ccm_values != "" ? var.hetzner_ccm_values : <<EOT
 networking:
   enabled: true
```

I think it's always safe to also set `clusterCIDR`:
```suggestion
networking:
 enabled: true
 clusterCIDR: "${var.cluster_ipv4_cidr}"
```